specification/loadtestservice/data-plane/Microsoft.LoadTestService/preview/2022-06-01-preview/loadtestservice.json

```diff
@@ -1380,8 +1323,8 @@
           }
         },
         "x-ms-examples": {
-          "TestRun_GetAppTestRunsSearch_MaximumSet_Gen": {
-            "$ref": "./examples/TestRun_GetAppTestRunsSearch_MaximumSet_Gen.json"
+          "GetAppTestRunsSearch": {
```

Comment:
We can change this - ListTestRunsSearch or something better

Reply:
Renamed the file